docs/docs/mapping/customizing_openapi_output.md

```diff
@@ -631,14 +631,105 @@ info:
   title: helloproto/v1/example.proto
   version: version not set
 consumes:
-- application/json
+  - application/json
 produces:
-- application/json
+  - application/json
 paths:
   /api/hello:
     get:
       operationId: EchoService_Hello
-...
+```
+
+### Disable default responses
+
+By default a 200 OK response is rendered for each service operation. But it is possible to disable this and explicitly define your service's responses, using the `disable_default_responses` option. Allowed values are: `true`, `false`.
```

Comment:
Could you just add a note somewhere to say that this does _not_ change the behavior of the gateway itself, and must be coupled with a custom ForwardResponseWriter that adjusts the response to be an accurate reflection of the API behavior? Maybe link to the docs section that talks about changing the response code.

Reply:
Added a note @johanbrandhorst